x-pack/libbeat/management/managerV2.go

```diff
@@ -865,6 +873,65 @@ func (cm *BeatV2Manager) reloadInputs(inputUnits []*agentUnit) error {
 	return nil
 }
 
+// reloadAPM reload APM tracing, it returns a bool and an error.
+// The bool, if set, indicates that the output reload requires an restart,
+// in that case the error is always `nil`.
```

Comment:
Comment seems out of date? `reloadAPM` doesn't return anything.

Reply:
Copy-and-paste error, will fix.